docs/topics/backend-details.rst

```diff
@@ -3,118 +3,23 @@
 Marketplace Backend Details
 ===========================
 
-marketplace-env
----------------
+`docker-compose` Commands
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Marketplace env is a library that:
+This is not a full list of commands, for that see the
+`docker-compose docs <https://docs.docker.com/compose/>`, just notable ones.
 
-* contains the fig configration, which is used to create docker containers
-* wraps fig and docker commands to make it easier to develop with
+.. function:: docker-compose up
```

Comment:
I would only recommend using: up -d since a data only container can kill all of docker-compose because its not very good with that.